File: scripts/audit/fixtures/PAN003-kind-segment-parity/bad/src/lib/kinds.ts
// Fixture (PAN003-kind-segment-parity / bad): TS kind→segment SoT that DISAGREES
// with the sibling scripts/lib/kinds.py (poem → "poems" here, but Python maps
// poem → "poetry") — the cross-language audit must fire.
type WorkKind = "book" | "poem" | "project";
type WorkSegment = "books" | "poems" | "projects";
export const SEGMENT_OF: Record<WorkKind, WorkSegment> = {
  book: "books",
  poem: "poems",
  project: "projects",
};

File: audit/fixtures/PAN003-kind-segment-parity/bad/pancratius/kinds.py
"""Fixture (PAN003-kind-segment-parity / bad): Python kind->segment SoT that
DISAGREES with the sibling src/lib/kinds.ts (poem -> "poetry" here, but the TS
maps poem -> "poems"). The audit execs this module, so it must import."""

SEGMENT_OF: dict[str, str] = {"book": "books", "poem": "poetry", "project": "projects"}
